src/widget/wtrackmenu.cpp

```diff
@@ -209,6 +218,14 @@ void WTrackMenu::createActions() {
         connect(m_pPurgeAct, &QAction::triggered, this, &WTrackMenu::slotPurge);
     }
 
+    if (featureIsEnabled(Feature::RemoveFromDisk)) {
+        m_pRemoveFromDiskAct = new QAction(tr("Delete Files From disk"), m_pRemoveFromDiskMenu);
```

Comment:
The lettering looks strange. All words start with an uppercase letter except the last one. "Delete Files From Disk" or "Delete Files from Disk" look more consistent and reasonable to me.

Reply:
"Delete Files From Disk" then.
looking at the other action titles for a pattern,.. it's a nice mix :) however, nouns should with upper case